crates/bindings-csharp/Runtime/Internal/Module.cs

```diff
@@ -258,14 +258,15 @@ Buffer args
             Runtime.Random = new((int)timestamp.MicrosecondsSinceEpoch);
             using var stream = new MemoryStream(args.Consume());
             using var reader = new BinaryReader(stream);
-            reducers[(int)id].Invoke(
-                reader,
-                new(
-                    Identity.From(caller_identity.Consume()),
-                    Address.From(caller_address.Consume()),
-                    timestamp.ToStd()
-                )
-            );
+            reducers[(int)id]
+                .Invoke(
+                    reader,
+                    new(
+                        Identity.From(caller_identity.Consume()),
+                        Address.From(caller_address.Consume()),
+                        timestamp.ToStd()
+                    )
+                );
```

Comment:
This seems to be an autoformatter change.
We might want to add a test to CI to validate that all C# checked in to the repo is formatted with some formatter? (in another PR).

Reply:
Yeah I want to add that too. I'm using CSharpier but this particular change happened because I upgraded their version locally. I'll pin its version & add CI in a separate PR.

Reply:
@kazimuth Here goes https://github.com/clockworklabs/SpacetimeDB/pull/1567